components/commons/metadata/mods.md

```diff
@@ -1,4 +1,8 @@
 | Modifiable Custom Properties              |
 | ----------------------------------------- |
+| `--mod-animation-duration-100`            |
+| `--mod-focus-indicator-gap`               |
+| `--mod-line-height-100`                   |
 | `--mod-overlay-animation-distance`        |
 | `--mod-overlay-animation-duration-opened` |
+| `--mod-sans-font-family-stack`            |
```

Comment:
This uncovers the fact that these previously unlisted mod names within `commons` may need to be renamed (not as part of this PR).

Reply:
Tell me more?

Reply:
These are in the _basebutton_ file I think. And the mod names are referencing the global token name which is generally something we have been avoiding.

Reply:
Oh yes, there's also nothing surfacing this information to consumers at this time. I wonder if we should be grepping the *compiled* css instead of src assets...

Reply:
> Oh yes, there's also nothing surfacing this information to consumers at this time. I wonder if we should be grepping the _compiled_ css instead of src assets...

Probably a good idea too. Currently, anything that uses an `@inherit` of some of those % prefixed "placeholder classes" may not have those mods listed. For example, modal should probably be showing `--mod-overlay-animation-duration-opened`.

Reply:
I've updated this to run on the _dist/index.css_ files with the latest commit.